### PR TITLE
Feat/param finalization

### LINKED ISSUE #1436: Actors v5 Crypto Economics updates
FILL IN as details become available

### PR DESCRIPTION
**Pull request by ZenGround0**
Closes #1436 #1433

### CODE REVIEW

**Review by arajasek — APPROVED** [review]
Seems reasonable to me

File: actors/builtin/miner/monies_test.go [hunk 1]
```diff
@@ -214,3 +214,29 @@ func TestPrecommitDepositAndInitialPledgePostiive(t *testing.T) {
...
+		atBalancePlusOneBaseFee := miner.AggregateNetworkFee(10, big.Sum(miner.BatchBalancer, builtin.OneNanoFIL))
+		atBalancePlusTwoBaseFee := miner.AggregateNetworkFee(10, big.Sum(miner.BatchBalancer, builtin.OneNanoFIL, builtin.OneNanoFIL))
+		atBalanceTimesTwoBaseFee := miner.AggregateNetworkFee(10, big.Mul(miner.BatchBalancer, big.NewInt(2)))
+
+		assert.True(t, atNoBaseFee.Equals(atBalanceMinusOneBaseFee))
+		assert.True(t, atNoBaseFee.Equals(atBalanceBaseFee))
+		assert.True(t, atBalanceBaseFee.LessThan(atBalancePlusOneBaseFee))
+		assert.True(t, atBalancePlusOneBaseFee.LessThan(atBalancePlusTwoBaseFee))
+		assert.True(t, atBalanceTimesTwoBaseFee.Equals(big.Mul(big.NewInt(2), atBalanceBaseFee)))
+	})
```
Comment (anorth) [inline actors/builtin/miner/monies_test.go:241]: I would like to see a couple of regression tests that code hard expected numbers expected out of AggregateNetworkFee. Like we have for block reward function (though less thorough is fine).

File: actors/runtime/runtime.go [hunk 2]
```diff
@@ -140,6 +140,9 @@ type Runtime interface {
 
 	// Note events that may make debugging easier
 	Log(level rt.LogLevel, msg string, args ...interface{})
+
+	// BaseFee returns the basefee value in attoFIL for the currently exectuting tipset.
```
Comment (anorth) [inline actors/runtime/runtime.go:144]:
```suggestion
	// BaseFee returns the basefee value in attoFIL per unit gas for the currently exectuting tipset.
```

File: actors/test/commit_post_test.go [hunk 3]
```diff
@@ -620,6 +625,76 @@ func TestAggregateOnePreCommitExpires(t *testing.T) {
...
+	// create miner
+	params := power.CreateMinerParams{
+		Owner:               addrs[0],
+		Worker:              addrs[0],
+		WindowPoStProofType: abi.RegisteredPoStProof_StackedDrgWindow32GiBV1,
+		Peer:                abi.PeerID("not really a peer id"),
+	}
+	ret := vm.ApplyOk(t, v, addrs[0], builtin.StoragePowerActorAddr, minerBalance, builtin.MethodsPower.CreateMiner, &params)
+	minerAddrs, ok := ret.(*power.CreateMinerReturn)
+	require.True(t, ok)
```
Comment (anorth) [inline actors/test/commit_post_test.go:647]: There's a `createMiner` helper for this now.

**Review by anorth — APPROVED** [review]

File: actors/builtin/miner/policy.go [hunk 4]
```diff
@@ -297,8 +297,8 @@ func RewardForDisputedWindowPoSt(proofType abi.RegisteredPoStProof, disputedPowe
 }
 
 const MaxAggregatedSectors = 819
-const MinAggregatedSectors = 1
-const MaxAggregateProofSize = 192000
+const MinAggregatedSectors = 4
+const MaxAggregateProofSize = 81960
```
Comment (magik6k) [inline actors/builtin/miner/policy.go:301]: Note to (likely) self -> write some higher-level test in lotus which posts maximum-size proof to the chain to make sure we don't have any length checks which could be too tight

**Review by magik6k — APPROVED** [review]

File: actors/builtin/miner/policy.go (hunk 4 above)
Comment (ZenGround0) [reply actors/builtin/miner/policy.go:301]: Great idea. FYI I made this 2x the size of the expected largest proof here to try to prevent problems

**Pull request merged**